incubator/kafka/README.md

```diff
@@ -175,3 +187,32 @@ such port at a time, setting the range at every Kafka pod is a reasonably safe c
 * There must not exist a service called `kafka` in the same namespace
 
 [brokerconfigs]: https://kafka.apache.org/documentation/#brokerconfigs
+
+## Prometheus Stats
+
+### Prometheus vs Prometheus Operator
+
+Prometheus is default option of this chart b/c the Prometheus Operator is part of another repo: 
```

Comment:
Standard Prometheus is the default monitoring option for this chart. This chart also supports the CoreOS Prometheus Operator, which can provide additional functionality like automatically updating Prometheus and Alert Manager configuration. If you are interested in installing the Prometheus Operator please see the [CoreOS repository for more information](https://github.com/coreos/prometheus-operator/tree/master/helm) or read through the [CoreOS blog post introducing the Prometheus Operator](https://coreos.com/blog/the-prometheus-operator.html)